### ISSUE TITLE
bug: Static files request triggers django session validation

### ISSUE THREAD

**Issue by introkun** [TreyWW/MyFinances#271]
### What version of MyFinances are you currently using?

v0.2.1

### What device type are you currently facing the issue on?

Desktop

### Describe the bug

Steps:
- HTTP Get request of /static/js/bundle.js

Actual result: DB call to check django session in debug logs
Expected result: static file is served w/o django session checks

```
[02/Apr/2024 00:05:07] "GET /static/js/bundle.js HTTP/1.1" 304 0
[DEBUG] (0.030) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2024-04-02T00:05:08.740441+00:00'::timestamptz AND "django_session"."session_key" = '9szn0g49xzk8yxbg79o5lq7vybrs4q54') LIMIT 21; args=(datetime.datetime(2024, 4, 2, 0, 5, 8, 740441, tzinfo=datetime.timezone.utc), '9szn0g49xzk8yxbg79o5lq7vybrs4q54'); alias=default
[DEBUG] (0.030) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2024-04-02T00:05:08.740441+00:00'::timestamptz AND "django_session"."session_key" = '9szn0g49xzk8yxbg79o5lq7vybrs4q54') LIMIT 21; args=(datetime.datetime(2024, 4, 2, 0, 5, 8, 740441, tzinfo=datetime.timezone.utc), '9szn0g49xzk8yxbg79o5lq7vybrs4q54'); alias=default
Session data corrupted
[WARNING] Session data corrupted
[02/Apr/2024 00:05:08] "GET /favicon.ico HTTP/1.1" 302 0
[DEBUG] (0.006) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2024-04-02T00:05:08.828222+00:00'::timestamptz AND "django_session"."session_key" = '123') LIMIT 21; args=(datetime.datetime(2024, 4, 2, 0, 5, 8, 828222, tzinfo=datetime.timezone.utc), '123'); alias=default
[DEBUG] (0.006) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2024-04-02T00:05:08.828222+00:00'::timestamptz AND "django_session"."session_key" = '123') LIMIT 21; args=(datetime.datetime(2024, 4, 2, 0, 5, 8, 828222, tzinfo=datetime.timezone.utc), '123'); alias=default
Session data corrupted
[WARNING] Session data corrupted
[02/Apr/2024 00:05:08] "GET /auth/login/?next=/favicon.ico HTTP/1.1" 200 16143
```

Maybe relates to #267.

**Comment by introkun** [comment]
On production, we should collect static files and serve via a webserver (nginx): `python manage.py collectstatic`
I'm not sure if we should fix it.

**Comment by TreyWW** [comment]
> On production, we should collect static files and serve via a webserver (nginx): `python manage.py collectstatic` I'm not sure if we should fix it.

I did used to have this, but it just used to crash production all the time as we use AWS Cloudfront to serve static files and it always had issues posting to that. I'm not sure how best we can allow users to "toggle" this on - maybe an if statement in the bash script using env vars?

**Comment by introkun** [comment]
Sorry, I don't think this is an issue. I see that we have "collectstatic", so it's only debug case which will not affect real users.

**Comment by introkun** [comment]
fixed in #312 for local django instance
